### PR TITLE
COMMERCE-9649 Fix order quantity constraints visibility

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-9649

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AddToCartTag.java [hunk 1]
```diff
@@ -82,6 +82,17 @@ public int doStartTag() throws JspException {
 				cpSku = _cpContentHelper.getDefaultCPSku(_cpCatalogEntry);
 				hasChildCPDefinitions = _cpContentHelper.hasChildCPDefinitions(
 					_cpCatalogEntry.getCPDefinitionId());
+
+				// In case of multiple SKUs get the first one to enable
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AddToCartTag.java:86]: remove comments

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
thank you @smottal

**Comment by andrea-ale-sbarra** [comment]
ci:reevaluate:1267401_5972

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
@smottal I re-sent this at #3212, can you please close this PR?
Thanks

**Pull request closed without merging**